Gems/Atom/RPI/Code/Source/RPI.Public/Culling.cpp

```diff
@@ -168,30 +171,81 @@ namespace AZ
             return worklistData;
         }
 
-        constexpr size_t WorkListCapacity = 5;
-        using WorkListType = AZStd::fixed_vector<AzFramework::IVisibilityScene::NodeData, WorkListCapacity>;
+        // Used to accumulate NodeData into lists to be handed off to jobs for processing
+        struct WorkListType
+        {
+            AZStd::vector<AzFramework::IVisibilityScene::NodeData> m_nodes;
+        };
+
```

Comment:
What's the reason to wrap this in a single member struct? Do we plan to add more members?

Reply:
Added a new variable to the struct